pytorch_lightning/callbacks/gpu_stats_monitor.py

```diff
@@ -199,3 +205,12 @@ def _get_gpu_device_stat_keys(self) -> List[Tuple[str, str]]:
             stat_keys.extend([('temperature.gpu', '°C'), ('temperature.memory', '°C')])
 
         return stat_keys
+
+    def _should_log(self, trainer) -> bool:
+        should_log = (
+            (trainer.global_step + 1) % trainer.row_log_interval == 0
+            or trainer.should_stop
+            or trainer.fast_dev_run
```

Comment:
should we do this with `fast_dev_run` too?

Reply:
Not sure, since user don't really interact with these callbacks during training. They only throw errors during construction and trainer init, so it should be save to not include these in `fast_dev_run`.

Reply:
Okay